MoviesApp/Services/MoviesAPI.swift

```diff
@@ -25,7 +24,7 @@ class MoviesAPIV1: MoviesAPI {
     
     convenience init(session: URLSession = URLSession(configuration:
         URLSessionConfiguration.default)) {
-        self.init(session: session, endPoint: "https://api.jsonbin.io/b/5e86bb0ef4dba3539aac2f2a")
+        self.init(session: session, endPoint: "http://api.jsonbin.io/b/5e86bb0ef4dba3539aac2f2a")
```

Comment:
<!--
  1 failure:  HTTPS should be f...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: MoviesApp/Services/MoviesAPI.swift;
  Line: 27;
-->
- :no_entry_sign: HTTPS should be favored over HTTP (`force_https`)